ibm/resource_ibm_container_cluster.go

```diff
@@ -952,6 +963,47 @@ func resourceIBMContainerClusterUpdate(d *schema.ResourceData, meta interface{})
 		}
 	}
 
+	if d.HasChange("labels") || d.IsNewResource() {
+		workerPoolsAPI := csClient.WorkerPools()
+		workerPools, err := workerPoolsAPI.ListWorkerPools(clusterID, targetEnv)
+		if err != nil {
+			return err
+		}
+		var poolName string
+		var poolContains bool
+
+		if len(workerPools) > 0 && workerPoolContains(workerPools, defaultWorkerPool) {
+			poolName = defaultWorkerPool
+			poolContains = true
+		} else if len(workerPools) > 0 && workerPoolContains(workerPools, computeWorkerPool) && workerPoolContains(workerPools, gatewayWorkerpool) {
+			poolName = computeWorkerPool
+			poolContains = true
+		}
+		if poolContains {
+			labels := make(map[string]string)
+			if l, ok := d.GetOk("labels"); ok {
+				for k, v := range l.(map[string]interface{}) {
+					labels[k] = v.(string)
+				}
+			}
+			err = workerPoolsAPI.UpdateLabelsWorkerPool(clusterID, poolName, labels, targetEnv)
+			if err != nil {
+				return fmt.Errorf(
+					"Error updating the labels%s", err)
```

Comment:
need a space after `labels` : `labels%s"` should be `labels %s"`

Reply:
Also, maybe I am wrong, but don't you use `%v` for errors?

Reply:
This followed the pattern in the rest of the code.